apps/finance/app/src/components/Transfers.js

```diff
@@ -175,6 +176,10 @@ const Transfers = React.memo(({ dao, tokens, transactions }) => {
       selectedDateRange.start ||
       selectedDateRange.end)
 
+  if (!transactions.length) {
```

Comment:
What do you think about moving this to `App` instead? That way we can also change the `transactions` prop in `Transfers` to be required.